spec/views/emails/email_form.html.erb_spec.rb

```diff
@@ -17,11 +21,17 @@
     expect(rendered).to have_field('email_content')
   end
 
-  it "renders email send button" do
+  it "renders email submit buttons" do
     expect(rendered).to have_button(I18n.t('emails.email_form.send'))
+    expect(rendered).to have_button(I18n.t('emails.email_form.save_template'))
+  end
+
+  it "renders copy recipients button" do
+    expect(rendered).to have_button(I18n.t('emails.email_form.copy'))
   end
 
   it "fills recipients_fills with set recipients" do
+    expect(rendered).to have_field('email_reply_to', with: @email.reply_to)
```

Comment:
Yes, but for the end-user it is kind of the same. Because he/she does not know of the internal email address and when you answer to an email you generally reply to the "Absender".